
package com.laegler.openbanking.soap.service;

import javax.xml.ws.WebFault;


/**
 * Too Many Requests
 *
 * This class was generated by Apache CXF 3.3.3
 * 2019-10-21T07:01:27.385+13:00
 * Generated source version: 3.3.3
 */

@WebFault(name = "GET_GetFundsConfirmationConsentsConsentId_429", targetNamespace = "http://laegler.com/openbanking/soap/model")
public class GETGetFundsConfirmationConsentsConsentId429 extends Exception {

    private java.lang.Object getGetFundsConfirmationConsentsConsentId429;

    public GETGetFundsConfirmationConsentsConsentId429() {
        super();
    }

    public GETGetFundsConfirmationConsentsConsentId429(String message) {
        super(message);
    }

    public GETGetFundsConfirmationConsentsConsentId429(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public GETGetFundsConfirmationConsentsConsentId429(String message, java.lang.Object getGetFundsConfirmationConsentsConsentId429) {
        super(message);
        this.getGetFundsConfirmationConsentsConsentId429 = getGetFundsConfirmationConsentsConsentId429;
    }

    public GETGetFundsConfirmationConsentsConsentId429(String message, java.lang.Object getGetFundsConfirmationConsentsConsentId429, java.lang.Throwable cause) {
        super(message, cause);
        this.getGetFundsConfirmationConsentsConsentId429 = getGetFundsConfirmationConsentsConsentId429;
    }

    public java.lang.Object getFaultInfo() {
        return this.getGetFundsConfirmationConsentsConsentId429;
    }
}
